packages/theme-predictive-search/__tests__/theme-predictive-search.test.js

```diff
@@ -0,0 +1,274 @@
+import "isomorphic-fetch";
+import fetchMock from "fetch-mock";
+import PredictiveSearch from "../src/theme-predictive-search";
+import searchAsYouTypeTheCallingFixture from "../__fixtures__/search_as_you_type_the_calling.json";
+import searchAsYouTypeTheFollowingFixture from "../__fixtures__/search_as_you_type_the_following.json";
+
+/* eslint-disable camelcase */
+const defaultConfig = {
+  search_as_you_type: {
+    fuzzy: true,
+    types: [PredictiveSearch.TYPES.PRODUCT]
+  }
+};
+
+describe("Search()", () => {
+  it("should be able to create an instance", () => {
+    const search = new PredictiveSearch(defaultConfig);
+    expect(search).toBeInstanceOf(PredictiveSearch);
+  });
+
+  describe("throws", () => {
+    it("when config object is not set", () => {
```

Comment:
Wonder if it's possible to have good defaults so we wouldn't need this. What do you think?